### PR TITLE
agnos 9

### LINKED ISSUE #28870: micd: PortAudioError
Happened a couple times in previous releases, with an uptick in 0.9.3-release with a couple dozen events.

https://commaai.sentry.io/issues/4061911255/?project=77924&referrer=release-issue-stream

### LINKED ISSUE #28880: Sound of Engaging and Disengagiong OpenPilot "Crashed"
### Describe the bug

During the drive of a long trip I recognized, that the Engage and Disengage sound of OpenPilot was gone.
All other sounds, still work (like Lane Departure...)

I had this issue one in the past, after reboot of my Comma3 the sound was there again.

### Provide a route where the issue occurs

6111bd7df5043394|2023-07-11--08-42-59--0

### openpilot version

0.9.3. Master

### Additional info

I use a clone of Master with one line difference in
[selfdrive/car/hyundai/hyundaican.py](https://github.com/commaai/openpilot/compare/master...DerGue69:openpilot:master#diff-b6a8111f5bf3fec6d90013111831d7d4355b7452da2ccc9d404d46118d819a59)

    # Some cars show an FCA fault when forwarding CF_Lkas_FusionState. Hide FCA Fault:
    values["CF_Lkas_FusionState"] = 0

[selfdrive/car/hyundai/hyundaican.py](https://github.com/commaai/openpilot/compare/master...DerGue69:openpilot:master#diff-b6a8111f5bf3fec6d90013111831d7d4355b7452da2ccc9d404d46118d819a59)

### PR DESCRIPTION
**Pull request by adeebshihadeh**
- [x] no pulse?
- [x] fix bootsplash blink
- [x] reduced banding when recording some device displays
- [x] purple prime connection
- [x] update setup
- [x] USB3 support in the ABL
- [x] serial number in the hostname, e.g. `tici` -> `comma-aeffe5d0`
- [x] package updates

closes #28870, closes #28880

---

# internal

## release checklist

- [x] openpilot CPU usage unchanged
- [x] wifi
- [x] modem
- [x] image size
- [x] sounds
- [x] `python` is our python, not system version
- [x] pyenv shims works
- [x] clean openpilot build
- [ ] factory reset
  - [ ] user-prompted reset
  - [ ] corrupt userdata
  - [x] finish ABL reset
- [ ] color calibration
  - [x] from /persist/comma/
  - [ ] directly from panel over sysfs
- [x] clean setup
- [x] cameras
- [x] update works on warm boot
  - [x] previous -> new
  - [x] new -> previous

### ABL

- [x] fastboot USB enumeration
- [x] system reset trigger works
- [x] system reset menu times out after 60s
- [x] boot time hasn't regressed (3.8s)

### XBL

- [ ] display init works in cold and hot temperatures
- [x] boot time hasn't regressed (2.4s)

#### networking

- [x] continue button disabled when no connection
- [x] forget/connect to wifi

#### custom URL
- (a) Not a real URL (e.g. `comma`, `abc123`, `...`)
  - [x] "Ensure the entered URL is valid"
  - [x] Start over
  - [x] Reboot device
- (b) Website but not an installer URL (e.g. `github.com`, `comma.ai`, `installer.comma.ai`)
  - [x] "No custom software found at this URL."
- (c) Valid installer URL (e.g. `openpilot.comma.ai`)
  - [x] Download successful (comma logo or installer appears)
  - [x] `/tmp/installer_url` should contain the installer URL

### CODE REVIEW

**Comment by devtekve** [comment]
Thoughts on including python 3.12 instead of 3.11.4?

**Comment by devtekve** [comment]
> Why?

Significant performance improvements and smaller footprint for what I can read around.. 

- [Faster Python: More Specializations and Inlined Comprehensions](https://realpython.com/python312-new-features/#faster-python-more-specializations-and-inlined-comprehensions )
- [Support for the Linux perf Profiler](https://realpython.com/python312-new-features/#support-for-the-linux-perf-profiler)
- [Low impact monitoring for CPython](https://docs.python.org/3/whatsnew/3.12.html#pep-669-low-impact-monitoring-for-cpython)
- [A Per-Interpreter GIL](https://docs.python.org/3/whatsnew/3.12.html#pep-684-a-per-interpreter-gil)

typehints improvements... among other things

**Comment by adeebshihadeh** [comment]
Cool. Want to make an openpilot PR to update it and get the tests passing? If that happens, we might be able to make it happen for this AGNOS, otherwise it'll probably happen in a future AGNOS.

**Comment by adeebshihadeh** [comment]
Hmm, I don't like that it's the `.0` release though. It's generally best to wait for a few bugfix releases.

**Comment by devtekve** [comment]
> Cool. Want to make an openpilot PR to update it and get the tests passing? If that happens, we might be able to make it happen for this AGNOS, otherwise it'll probably happen in a future AGNOS.

Sure, I can give it a try, do the test honor pyproject.toml for the python env? or do they need to be updated as well?

File: RELEASES.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Version 0.9.6 (2023-XX-XX)
 ========================
+* AGNOS 9
```
Comment (Hap456) [inline RELEASES.md:4]: agnos9

**Pull request merged**